admin/app/controllers/metrics/MetricsController.scala

```diff
@@ -49,16 +49,6 @@ class MetricsController(
     } yield NoCache(Ok(views.html.lineCharts(googleBot404s, Some("GoogleBot 404s"))))
   }
 
-  def renderMemory(): Action[AnyContent] = Action.async { implicit request =>
-    for {
-      metrics <- MemoryMetrics.memory()
-    } yield NoCache(Ok(views.html.lineCharts(metrics)))
-  }
-
-  def renderAssets(): Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(NoCache(Ok(views.html.staticAssets(AssetMetricsCache.sizes))))
-  }
-
```

Comment:
I am fine with removing this functionality. However we should also remove the code that generate those assets metrics (https://github.com/guardian/frontend/blob/master/dev/teamcity/dist-assetmonitor-tc)

Reply:
@sndrs can we remove the asset-monitor build step from team city?